test/unit/test_dss_docker_image.py

```diff
@@ -75,39 +75,49 @@ def test_entrypoint_default(facts):
 
 
 def test_entrypoint_with_copy_args():
-    jupyter = "/home/jupyter/jupyterenv/bin/jupyter-lab"
-    port = "port"
-    entrypoint = "/path/to/entrypoint.py"
-    initial = "/path/to/initial"
-    final = "/path/to/final"
-    user = "jupyter-user-name"
-    password = "jupyter-default-password"
-    logfile = "/path/to/jupyter-server.log"
     facts = {
         "dss_facts": {
```

Comment:
hm, slowly, I am asking me, if we should create a pydantic model for this json for validation

Reply:
I will create a ticket.

Reply:
https://github.com/exasol/ai-lab/issues/245, already linked to roadmap